src/pages/authPage/auth-page.css

```diff
@@ -0,0 +1,91 @@
+@import "/src/styles.css";
+
+.page-wrap {
+    grid-template-areas:
+        'p_nav p_nav p_nav p_nav p_nav p_nav'
+        'p_content p_content p_content p_content p_content p_content'
+        'p_content p_content p_content p_content p_content p_content';
+}
+
+.page-sidebar {
+    display: none;
+}
+
+.page-footer {
+    display: none;
+}
+
+.main-content {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+}
+
+
+/*TODO: move to component library */
+.dui-auth-card {
+    width: fit-content;
+    height: fit-content;
+    padding: 2rem;
+    padding-left: 7.5rem;
```

Comment:
use shorthand property for padding and margin i.e Top,Right,Bottom,Left

Reply:
will be doing it as part of refactoring.